test/e2e/density.go

```diff
@@ -186,6 +209,108 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			if itArg.runLatencyTest {
+				Logf("Schedling additional Pods to measure startup latencies")
+				// Use service to assure even spread of pods between nodes
```

Comment:
It shouldn't change anything. If there is heavy clustering of pods it's probably a bug, I'd expect some imbalance with round robin scheduling but not too much.